smartseq2_single_sample/build_references/build_hisat2_reference/BuilHISAT2Referencebundle.wdl

```diff
@@ -0,0 +1,37 @@
+task BuildHISAT2reference{
+  String ref_name
+  String gtf_version
+  String dbsnp_version
+
+  command {
+    /opt/tools/hisat2-2.1.0/make_grch38_snp_tran_gencode.sh ${gtf_version} ${dbsnp_version} 
```

Comment:
I would ideally like to have the docker install the scripts into the path so that users can more easily use the packaged scripts and binaries.

Reply:
it need to be attached to HISAT2